gym_extensions/discrete/classic/pendulum_contextual.py

```diff
@@ -51,3 +49,24 @@ def change_context(self, context_vector):
         self.context    = context_vector
         self.max_speed  = self.context[0]
         self.max_torque = self.context[1]
+
+
+    def context_space_info(self):
+        context_info_dict = {}
+
+        context_info_dict['context_vals'] = {
+                                            'max_speed':self.max_speed, 
+                                            'max_torque':self.max_torque
+                                            }
+
+        context_info_dict['context_high'] = self.context_high
+        context_info_dict['context_low' ] = self.context_low
+        context_info_dict['state_dims'  ] = len(self.state)
+        
+        # yes, I don't care that there are two/three different actions here; for the contextual setup
+        # I need to know what the size of the action vector I need to pass to the transition function
+        context_info_dict['action_dims' ] = 1
```

Comment:
In action space the action_dims = 2, while in my settings I need the outputDims for the action, so I was always taking (action_dims -1); in the same time, since I need both - state_low/high and context_low/high, I think getting all the parameters from one place would be more clear for everybody.